packages/cozy-client/src/CozyClient.js

```diff
@@ -863,12 +863,18 @@ class CozyClient {
    * Get a query from the internal store.
    *
    * @param {String} id - Id of the query (set via Query.props.as)
+   * @param {Boolean} options.hydrated - Whether documents should be returned already hydrated (default: false)
```

Comment:
```suggestion
   * @param {Boolean} options.hydrated - Whether documents should be returned already hydrated (default: false)
      We can have already hydrated documents when we get them from the local store instead of Couchdb
```

Reply:
Mmh it seems a bit confusing to put this here since getQueryFromState is always taking data from the local store.

Reply:
Do you have a better way to explain when the swith is used and why ? I would welcome some context in comments

Reply:
"If you want to have relationships resolved when getting data from the store, you should toggle this flag" ?

Why wouldn't you want the data to be always hydrated ? Maybe you want to directly touch raw relationships ? The relationship resolving mechanic has a performance hit since it need to find all related documents in the store for each documents of the query.

Since this is a change a behavior from getQueryFromState I preferred to have an opt-in flag here but this could be rediscussed.

Reply:
Oh! it seems I understood wrongly what you did, sorry.

👌 for me